### PR TITLE
Refreshing chapter 2

### PR DESCRIPTION
**Pull request by alabd14313**

### CODE REVIEW

**Comment by alabd14313** [comment]
Hello @mikepan !
Please don't merge @Momel15's PR. because I want refreshing some parts (From Texturing to the end of this chapter).
It is good help if @Momel15 convert images of chapter 3 to 10, before I start refining them.

**Comment by alabd14313** [comment]
Chapter 2 now is ready for releasing. Also I want to refresh it more and more.

File: text/02-FirstGame.md [hunk 1]
```diff
@@ -97,151 +111,94 @@ To play the game inside Blender, open the file game.blend. This file is only a p
...
-\Book\Chapter02\game\_final\-The final as of the end of this exercise; use for reference.
+- **Book/Chapter02/game\_final** -The final as of the end of this exercise; use for reference.
 
-\Book\Chaper02\references\-Files to support the making of the game.
+- **Book/Chaper02/references** -Files to support the making of the game.
 
 For the rest of the chapter, we will refer to the files from your //game\_my/ top folder.
 
+
 ## Modeling
```
Comment (alabd14313) [inline text/02-FirstGame.md:125]: Although I added more images, Modeling needs more clarification and images. There are some gaps between modeling steps.

**Comment by mikepan** [comment]
Done!

**Pull request merged**